client/flutter/helper-scripts/flutter-netlify-build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+FLUTTER_BRANCH=`grep channel: .metadata | sed 's/  channel: //g'`
+FLUTTER_REVISION=`grep revision: .metadata | sed 's/  revision: //g'`
+
+git clone https://github.com/flutter/flutter.git
```

Comment:
I'm working on this PR now, i'll finish it up :)

Reply:
I want our generic builds to be at stable, but for the web build we need beta. 
Note that stability of the web build is not the most important thing since these are just previews.